Sources/MintKit/MintKit.swift

```diff
@@ -91,21 +91,25 @@ public struct Mint {
             return
         }
 
-        if !package.checkoutPath.exists {
-            print("🌱  Cloning \(package.git)...")
-            do {
-                try shellOut(to: "git clone \(package.git) \(package.checkoutPath.lastComponent)", at: package.path.string)
-            } catch {
-                throw MintError.repoNotFound(package.git)
-            }
-        }
+        let checkoutPath = Path.temporary + "mint"
```

Comment:
@SDGGiesbrecht I assume `NSTemporaryDirectory()` which is what `PathKit.temporary` uses is suitable on Linux?

Reply:
@yonaskolb, Probably, but Linux does not work [at all](https://travis-ci.org/yonaskolb/Mint/jobs/306167062#L679), and would probably require swapping out dependencies, so I would not even worry about it.

Note: `NSTemporaryDirectory()` is on the road to deprecation as `Foundation` moves away from paths in favour of the `URL`‐based API. If that matters to you, you can take a peek [here](https://github.com/SDGGiesbrecht/SDGCornerstone/blob/223f17b34b8bcb44141b120a0c968a20ff8ca40f/Sources/SDGCornerstone/Persistence/FileManager.swift#L49).